osu.Game/Skinning/Editor/SkinSelectionHandler.cs

```diff
@@ -215,14 +229,16 @@ private static void updateDrawablePosition(Drawable drawable, Vector2 screenSpac
                 drawable.Parent.ToLocalSpace(screenSpacePosition) - drawable.AnchorPosition;
         }
 
-        private void applyOrigin(Anchor anchor)
+        private void applyOrigins(Anchor origin)
         {
             foreach (var item in SelectedItems)
             {
                 var drawable = (Drawable)item;
 
+                if (origin == drawable.Origin) continue;
```

Comment:
I don't know how necessary this and...
https://github.com/ppy/osu/blob/10b6b7290923a112dfee58c8a8c820dc32934611/osu.Game/Skinning/Editor/SkinSelectionHandler.cs#L322
... are, but I wanted them to at least be consistent. In the case that some drawables already have the correct anchor/origin, it's possible that by moving them slightly it'll cause some jitter. Probably sub-pixel, though.

Reply:
I'm not sure where the "pixel" or "subpixel" part comes from. We are using floats, which are never going to be "pixel" based, nor are they ever round to `int` values or anything.